README.md

```diff
@@ -23,9 +23,9 @@
   </a>
 </p>
 
-**semantic-release** automates the whole package release workflow including: determining the next version number, generating the release notes and publishing the package.
+**semantic-release** automates the whole package release workflow including: determining the next version number, generating the release notes, and publishing the package.
 
-This removes the immediate connection between human emotions and version numbers, strictly following the [Semantic Versioning](http://semver.org) specification.
+This removes the immediate connection between human emotions and version numbers, strictly following the [Semantic Versioning](http://semver.org) specification and communicating the **impact** of changes to consumers.
```

Comment:
impact of changes from the perspective of package consumers is the most common point that i reiterate with teams new to semver and semantic-release in order to help with understanding. i think it would be helpful if we highlight that point more in our documentation. do you agree?